I wish_If only/maria_ptitsyna.md

```diff
@@ -0,0 +1,10 @@
+1. If only you had come earlier to check out an architecture's lameness suggested of a freshman.
+2. I wish we had waited until his arrival, otherwise we would be in touch with a current project's stage and what we need to do next time. 
+3. I wish we had met with you before leave and discussed your quit reason, I have a lot of thoughts about it and I would like to share them with you.
+4. I regret finishing the project that hadn't been started because of lack of assistants and a consisting 	finance. If it had been fine and dandy, the idea would definitely have been successful on the market and stayed a new blockbuster.
+5. He wishes he hadn't come and led the friend. If I were him, I wouldn't do that. If he had come on time, it wouldn't have happened.
+6. But for you, I don't know what I would answered the customer, you took a conversation in the different way and we managed to avoid a serious conversation about pressing deadlines.
+7. I wish I could read this book in original, so I could find out about the new tendency in the developer's world from the source, don't wait for the years till it will be translated.
```

Comment:
in the original - set phrase
not waiting for years until it is translated  - till/until in this case no Future as they introduce subordinate clause of time